README.md

```diff
@@ -60,7 +60,11 @@ For advanced users interested in keeping more up-to-date on project development
 
 ### 1.2 Obtaining supporting data
 
+<<<<<<< HEAD
```

Comment:
@yihungkuo 
```
<<<<<<< HEAD

============

>>>>>>>> <commit hash>
```
Indicates that there are unresolved merge conflicts. You need to select the text you want above or below the `===========` line (or a combination of the two if there are parts of both you want to retain), and remove the `<<<<<, =====, >>>>>` lines.

Reply:
@wrongkindofdoctor and @jkrasting I am confused. I got these from the latest GFDL develop branch just a few hours ago, and these were already there. Could you confirm? And is there any thing I need to do?

Reply:
These lines are not in the develop branch of NOAA-GFDL fork, but they are in your fork, which means that the conflicts occurred when you updated your copy of the develop branch.

Reply:
Got it. Fixed.